src/Cardano/Wallet/Mnemonic.hs

```diff
@@ -0,0 +1,332 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- | Module provides mnemonic creation and
+-- | restoring from backup phrase functionality
+
+module Cardano.Wallet.Mnemonic
+       (
+       -- * Types
```

Comment:
We should avoid variable length indentation here, and use a fixed-width indentation here (4 characters).

https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#avoid-variable-length-indentation

Reply:
Agreed, let's indent module exports by 4 chars only :+1: